cookbooks/cdo-nginx/templates/default/nginx.conf.erb

```diff
@@ -0,0 +1,53 @@
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+
+events {
+  worker_connections 1024;
+  use epoll;
+}
+
+http {
+  sendfile on;
+  tcp_nopush on;
+  tcp_nodelay on;
+  types_hash_max_size 2048;
+
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  access_log /var/log/nginx/access.log;
+  error_log /var/log/nginx/error.log;
+  proxy_redirect off;
+  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+  client_max_body_size 4G;
+  keepalive_timeout 5;
```

Comment:
FWIW many tuning examples on the web seem to use longer values (10-30 seconds.)  Nginx will close keep-alive connections if it is running out of connections.